docker-compose.yml

```
@@ -106,6 +106,7 @@ services:
   memcached:
     <<: *restart_policy
     image: "memcached:1.6.9-alpine"
+    command: "-I 25M"
```

Better to use the array notation?
```suggestion
    command: ["-I", "25M"]
```